site/docs/v1/tech/core-concepts/_email-templates.adoc

```diff
@@ -0,0 +1,44 @@
+[.api]
+[field]#Email update# [optional]#Optional# [since]#Available since 1.30.0#::
+The email template to use when notifying a user that their email address has been updated.
+
+ifeval::["{page}" == "application"]
```

Comment:
These two fields, `Email Verification` and `Email Verified` are referred to within the "Email verification settings" section on tenant and handled separately there.